via/static/index.html

```diff
@@ -4,12 +4,13 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>via.hypothes.is</title>
-
-  <link rel="stylesheet" href="/vendor/unpkg.com/normalize.css" />
+  <link rel="icon" href="favicon.ico" type="image/x-icon" />
   <style>
     body {
       font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
       height: 100vh;
+      padding: 0;
+      margin: 0;
```

Comment:
This prevents the scroll bar (in Chrome at least) and appears to be all normalise was doing for us. Although maybe it explodes on Firefox OS or something?